external-builds/pytorch/check_venv.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+if [ -n "${VIRTUAL_ENV}" ]; then
+	echo "virtual env set"
+	VENV_DIR_ENV_VARIABLE=$(realpath ${VIRTUAL_ENV})
+	THEROCK_VENV_DEF_DIR=$(realpath "../../.venv")
+
+	echo "VENV_DIR_ENV_VARIABLE: ${VENV_DIR_ENV_VARIABLE}"
+	echo "THEROCK_VENV_DEF_DIR: ${THEROCK_VENV_DEF_DIR}"
+	if [ "${VENV_DIR_ENV_VARIABLE}" == "${THEROCK_VENV_DEF_DIR}" ];
+	then
+		echo "python virtual env: ${VIRTUAL_ENV}"
+	else
+		echo "Python VIRTUAL_ENV: ${VIRTUAL_ENV}"
+		echo "THEROCK default Python virtual env not active: ${THEROCK_VENV_DEF_DIR}"
+	fi
+else
+	echo "virtual env not set"
+	if [ -f ../../.venv/bin/activate ]; then
```

Comment:
Use script relative paths, not pwd relative. Rely on `set +e` or `if` error handling for the `source` vs doing a check.